draft-ietf-httpbis-digest-headers.md

```diff
@@ -138,19 +137,20 @@ HTTP/1.1 ([RFC7231], Appendix B) obsoleted it:
   > inconsistently implemented with respect to partial responses.
 
 [RFC3230] provided a more flexible solution introducing the concept of
-"instance", and the header fields `Digest` and `Want-Digest`.
+"instance", and the fields `Digest` and `Want-Digest`.
 
 ## This Proposal
 
-The concept of `selected representation` defined in [RFC7231] made [RFC3230]
-definitions inconsistent with the current standard. A refresh was then required.
+The concept of `selected representation` defined in Section 6 of
+[I-D.ietf-httpbis-semantics] makes [RFC3230] definitions inconsistent with
```

Comment:
I would use `{{!SEMANTICS=I-D.ietf-httpbis-semantics}}`.  But that's just me.  I think that you can do something with the reference above to get a nicer symbolic reference as well, but I forget the syntax.

Reply:
I take this on board. One problem I encountered was how to reference the core document in the abstract, which can't be a real reference othereise it triggers idnits IIUC

I inclined to modify reference style separately in a single pass PR..

Reply:
Once we close this PR we can merge Martin's suggestion in a separate PR or in the final commit